### PR TITLE
Move font-face declarations to a separate CSS file

### PR DESCRIPTION
**Pull request by marcaaron**
### Details
Moving the fonts that load in `index.html` to a separate css file so they can be used on other pages.

### Fixed Issues
No Issue.

### Tests
1. Load web and desktop
2. Build web
3. Verify the fonts load on all platforms

### Tested On

- [x] Web
- [ ] Mobile Web
- [x] Desktop
- [ ] iOS
- [ ] Android

### Screenshots
❌

### CODE REVIEW

**Comment by marcaaron** [comment]
@roryabraham didn't make an issue for this. Doing some prep related to implementing Storybook where we can import the css file.

File: web/index.html [hunk 1]
```diff
@@ -3,70 +3,8 @@
...
-        * {
-            -webkit-font-smoothing: antialiased;
-            -moz-osx-font-smoothing: grayscale;
-        }
-
-        strong {
-            font-weight: 600;
-        }
-
         html,
```
Comment (roryabraham) [inline web/index.html:8]: NAB do you know what's the deal with these other styles / why we wouldn't move them to a css file too?

**Review by roryabraham — APPROVED** [review]
Not totally clear on why this is necessary but it tested fine and there's no harm in moving some css out of the html file, so LGTM

File: web/index.html (hunk 1 above)
Comment (roryabraham) [reply web/index.html:8]: NAB so if there's a reason you left those styles in here then feel free to self-merge.

Comment (marcaaron) [reply web/index.html:8]: Yep I only wanted to pull out things related to fonts here. When this is re-used we won't want to apply those `html` block styles or height 100% stuff.

**Pull request merged**